mne_bids/tests/test_write.py

```diff
@@ -781,7 +801,7 @@ def test_edf(_bids_validate):
     assert 'space-mri' in coordsystem_fname
     with open(coordsystem_fname, 'r') as fin:
         coordsystem_json = json.load(fin)
-    assert coordsystem_json['iEEGCoordinateSystem'] == 'Other'
+    assert coordsystem_json['iEEGCoordinateSystem'] == 'other'
```

Comment:
The `other` keyword should be with a capital `O` according to the BIDS spec: https://bids-specification.readthedocs.io/en/stable/99-appendices/08-coordinate-systems.html#coordinate-systems-applicable-to-meg-eeg-and-ieeg

this probably has to be fixed in some part of the code?

Reply:
Done